exercises/basic_filter/problem.md

```diff
@@ -27,10 +27,12 @@ The function should return an array containing the messages themselves, *without
 ## Example
 
 ```
-[ 'Tempor quis esse consequat sunt ea eiusmod.',
-  'Id culpa ad proident ad nulla laborum incididunt.',
-  'Ullamco in ea et ad anim anim ullamco est.',
-  'Est ut irure irure nisi.' ]
+[ { message: 'Exercitation irure fugiat anim incididunt sunt duis deserunt laboris.' },
+  { message: 'Eiusmod ipsum esse ullamco anim labore tempor eu nostrud commodo consequat consectetur amet.' },
+  { message: 'Mollit ea labore do laboris nostrud proident ullamco.' },
+  { message: 'Laboris excepteur aliquip ullamco culpa.' },
+  { message: 'Culpa minim proident dolor ipsum ullamco est qui non aute aliquip excepteur sunt.' },
+  { message: 'Cupidatat duis sit aute pariatur amet sit excepteur cupidatat.' } ]
```

Comment:
This is incorrect output though, the solution expects an array of strings, not an array of objects.